doc/tasks/readme.md

```diff
@@ -0,0 +1,737 @@
+Per GUI-Button kann das Inventar wieder geschlossen werden.
+
+### Item aus Container-Inventar in Charakter-Inventar transferieren
+
+Bei geöffnetem Container-Inventar kann ein Item aus dem Container-Inventar ausgewählt werden und in das Inventar
+des Spielcharakters transferiert werden. Per GUI-Button können auch alle Items aus dem Container-Inventar in das
+Inventar des Spielcharakters übertragen werden.
+
+### Item aus Charakter-Inventar in Container-Inventar transferieren
+
+Bei geöffnetem Spielcharakter-Inventar kann ein Item aus dem Spielcharakter-Inventar ausgewählt werden und in das Inventar
+eines Containers transferiert werden.
+
+### Ein Item aufheben
+
+Ein Item, welches auf dem Boden des Dungeons liegt, kann vom Spielcharakter aufgesammelt werden, indem der Spielcharakter
+mit der Hitbox des Items kollidiert.
+
+### Informationen über ein Item ansehen
+
+Bei geöffnetem Inventar (unabhängig zu welcher Entität das Inventar gehört), können Informationen über ein
+Item innerhalb des Inventars angezeigt werden.
+
+### Das Inventar des Spielcharakters öffnen
+
+Per Tastendruck kann das Inventar des Spielcharakters geöffnet und geschlossen werden.
+
+### Ein Item auf den Boden fallen lassen
+
+Bei geöffnetem Inventar (unabhängig zu welcher Entität das Inventar gehört), kann ein ausgewähltes Item aus dem Inventar
+entfernt und auf den Boden gefallen lassen werden.
+
+### Eine Waffe aus dem Inventar ausrüsten
+
+Bei geöffnetem Spielcharakter-Inventar kann eine Waffe als aktive Waffe des Spielcharakters ausgerüstet werden.
+
+### Ein Item aus dem Inventar nutzen
+
+Bei geöffnetem Spielcharakter-Inventar kann ein Item genutzt werden (sofern es ein nutzbares Items ist), sodass ein
+"Nutzungseffekt" (bspw. Heilung der Gesundheit des Spielcharakters) eintritt, welcher sich für die Items unterscheiden kann.
+
+### Den Text einer Schriftrolle lesen
+
+Bei geöffnetem Inventar kann eine Schriftrolle aus dem Inventar ausgewählt und gelesen werden, wofür eine separater
+GUI-Dialog geöffnet wird, der den Text der Schriftrolle enthält.
+
+### Eine Entität im Dungeon "inspizieren"
+
+Entitäten im Dungeon können vom Spielcharakter inspiziert werden, wodurch sich ein GUI-Dialog öffnet, der Informationen
+über die inspizierte Entität anzeigt.
+
+### Ein Monster angreifen
+
+Der Spielcharakter kann mit einer ausgerüsteten Waffe ein Monster angreifen und dessen Gesundheit Schaden zufügen.
+
+### Schaden von einem Monster zugefügt bekommen
+
+Der Gesundheit des Spielcharakters kann durch ein Monster Schaden zugefügt werden.
+
+### Monster besiegen
+
+Wenn die Gesundheit eines Monsters auf 0 fällt, wird es besiegt und stirbt, wodurch es ggfs. Erfahrungspunkte und
+Items aus seinem Inventar (falls vorhanden) fallen lässt.
+
+### Einen Block verschieben
+
+Per Interaktion kann der Spielcharakter einen Block im Dungeon in eine bestimmte Richtung verschieben.
```

Comment:
"Block" ist noch nicht definiert. 

Und muss es ein Block sein? Könnte das nicht auch ein Item oder ein NPC sein?

Reply:
Würde aus Block => "eine andere Entität" machen

Reply:
Ist jetzt definiert

Reply:
> Würde aus Block => "eine andere Entität" machen

Finde ich gut, und das passt auch technisch zu dem, was wir im Game treiben.

Reply:
Ist das dann resolved?

Reply:
Irgendwie spinnt Github. Ich hab per Mail einen Kommentar von @malt-r bekommen: "_Ist das dann resolved?_", finde den aber nicht hier wieder (auch nicht, wenn ich auf den Link in der Mail gehe).

@malt-r Im Prinzip ist das "resolved", da Du nun weiter oben "Block" als "Entität" definiert hast. Ich schreibe aber absichtlich "im Prinzip", weil ich noch nicht verstehe, wieso Du an "Block" festhälst und nicht einfach "Entität" nutzt. An anderen Stellen nutzen wir ja auch die Begrifflichkeit aus dem Game, warum nicht hier auch?

Reply:
Das würde für mich implizieren, dass wir jede Entität mit dem Spielcharakter verschieben können, falls wir das so allgemein formulieren, was ja prinzipiell schwierig ist. Daher sehe ich das als sinnvoll an, einen extra Entitätstyp zu definieren, der das explizit erlaubt. Ob der Typ unbedingt "Block" heißen muss, weiß ich aber auch nicht.

Reply:
> "Eine Entität verschieben": Per Interaktion kann der Spielcharakter eine Entität im Dungeon in eine bestimmte Richtung verschieben.

Da würde ich jetzt nicht rauslesen, dass ich _immer jede_ Entität verschieben kann. Und letztlich sind unsere "lebenden" Sachen im Dungeon Entitäten. 

Ok, vielleicht möchte ich auch Wände oder Türen verschieben. Das wären keine Entitäten. Dann vielleicht "Ein Spielelement verschieben"? Dann bleibt es offen, was genau man schiebt?

@AMatutat Wie siehst Du das? Wir sind ja so halb auf dem Weg, alles interagierbare als Entität zu betrachten. D.h. verschiebbare Wände und Türen müsste man als Entität modellieren, um damit interagieren zu können. Sollten wir hier lieber von "Entität" oder besser von "Spielelement" sprechen? "Block" finde ich zu einschränkend.

Reply:
Technisch gesehen "kannst" du nur Entitäten verschieben.
Wände und Türen sind (aktuell) keine Entitäten und daher nicht verschiebbar. 
=> Würde beim Begriff Entität bleiben. Würde jetzt aber nicht daraus lesen, dass ich gleich jede Entität verschieben kann. Ich kann ja auch nicht jede Entität verkloppen

Reply:
@malt-r Sieht so aus, als ob wir ein gemeinsames Verständnis haben. Dann sollte es wirklich "Entität" heissen und "Block" verschwindet.

Reply:
> Ich kann ja auch nicht jede Entität verkloppen

Deswegen sprechen wir da ja auch von Monstern, nicht von Entitäten. Aber okay, ich beuge mich der Mehrheit :)

Reply:
> > Ich kann ja auch nicht jede Entität verkloppen
> 
> Deswegen sprechen wir da ja auch von Monstern, nicht von Entitäten. 

Guter Punkt. Sollten wir dann auch die "Monster" durch "Entität" ersetzen? Wir wissen hier eh nicht, was genau ein "Monster" ist - aber wir kennen "Entität" aus technischer Sicht.